### PR TITLE
docs/release-notes: update for release

### PR DESCRIPTION
**Pull request by prestist**

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Major changes:
 Minor changes:
 
 
+Packaging changes:
+
+
+## Upcoming Afterburn 5.4.2 (2023-05-18)
```
Comment (bgilbert) [inline docs/release-notes.md]:
```suggestion
## Afterburn 5.4.2 (2023-05-18)
```

**Review by bgilbert — COMMENTED** [review]
As discussed, we should also mention the regression fix on Azure with `openssl` &ge; 0.10.46.

File: docs/release-notes.md (hunk 1 above)
Comment (prestist) [reply docs/release-notes.md]: Oh my goodness. (sorry)

File: docs/release-notes.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ Minor changes:
 
 Packaging changes:
 
+
+## Afterburn 5.4.2 (2023-05-18)
+
+Packaging changes:
+
+- Migrate from `parse` to `parse2`; fix regression for Azure _openssl 0.10.46+_
```
Comment (bgilbert) [inline docs/release-notes.md]: https://github.com/coreos/afterburn/pull/926

**Review by bgilbert — APPROVED** [review]

**Comment by bgilbert** [comment]
The remaining CI doesn't test this.

**Pull request merged**